.travis.yml

```diff
@@ -21,4 +21,8 @@ script:
   - cabal build
   - cabal test
   - cabal install
+  - ([ "$GHCVER" = "7.6.3" ] || cabal configure --enable-tests --enable-executable-dynamic)
+  - ([ "$GHCVER" = "7.6.3" ] || cabal build)
+  - ([ "$GHCVER" = "7.6.3" ] || cabal test)
+  - ([ "$GHCVER" = "7.6.3" ] || cabal install)
```

Comment:
Yeah, we would want to test both.

I would also assume that `build` and `test` are redundant, since the first happens anyway and AFAIK the latter is unaffected by the dynamic executable stuff (or is it?).